bin/update_folders.py

```diff
@@ -112,9 +112,11 @@ def update_folders(
     folder_name = sys.argv[3]
     file_name = sys.argv[4]
 
-    update_folders(
-        project_name=project_name,
-        submission_id=submission_id,
-        folder_name=folder_name,
-        input_file=file_name
-    )
+    # Updating the subfolders with non-INVALID output files, and docker logs.
+    if "INVALID" not in file_name:
```

Comment:
I think this is fine, but do you think it would be better if the name checking happened in the `script` section of `UPDATE_FOLDERS.nf`? That way the Python script doesn't need to be executed at all.

Reply:
Modifying the process instead of the Python script feels cleaner to me, too. Updated